controllers/ovncontroller_controller.go

```diff
@@ -611,7 +642,7 @@ func (r *OVNControllerReconciler) reconcileNormal(ctx context.Context, instance
 	}
 
 	// create OVN Config Job - start
-	if instance.Status.NumberReady == instance.Status.DesiredNumberScheduled {
+	if instance.Status.NumberReady == instance.Status.DesiredNumberScheduled && instance.Status.OVSNumberReady == instance.Status.DesiredNumberScheduled {
```

Comment:
just a question - do we need to wait for both "NumberReady" and "OVSNumberReady" to be as desired? If I'm not mistaken technically it should be enough if we will have OVS PODs ready to run config job and set config knobs in the ovs database, right?

Reply:
yeah, make sense. ConfigJob will run init.sh that basically write config on OVS DB.